pkgs/top-level/perl-packages.nix

```diff
@@ -2353,6 +2353,20 @@ let self = _self // overrides; _self = with self; {
     buildInputs = [ PathClass TryTiny ];
   };
 
+  CryptX = buildPerlModule rec {
+    name = "CryptX-0.025";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/M/MI/MIK/${name}.tar.gz";
+      sha256 = "f8b7e3ec1713c8dfe3eef9d114f45f223b97e2340f81a20589b5605fa49cfe38";
+    };
+    buildInputs = [ JSON ];
```

Comment:
If you need JSON as a propagted build input it shouldn't be necessary to put it as a normal build input as well.